api/src/orgs/schema.js

```diff
@@ -32,6 +32,8 @@ export default `
     nodes: [Organisation]
     pageInfo: PageInfo!
     totalCount: Int!
+    totalAUM: Float!
+    totalActivity: Float!
```

Comment:
Since these are not relay-specific we should attach comments for the docs generated by introspection queries. The comments should be above the properties and briefly explain what the properties convey.